AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessorTests.swift

```diff
@@ -97,6 +100,22 @@ class ItemListProcessorTests: AuthenticatorTestCase {
         XCTAssertEqual(item.totpKey, String.base32Key)
     }
 
+    /// Tests that the `itemListCardState` is set to `none` if the card has been closed.
+    func test_determineItemListCardState_closed_download() async {
+        configService.featureFlagsBool = [.enablePasswordManagerSync: true]
+        application.canOpenUrlResponse = false
+        await subject.perform(.closeCard(.passwordManagerDownload))
+        XCTAssertEqual(subject.state.itemListCardState, .none)
+    }
+
+    /// Tests that the `itemListCardState` is set to `none` if the card has been closed.
```

Comment:
⛏️ Description of the func are the same. They should denote which card they are closing.